src/main/java/com/snowflake/kafka/connector/internal/metrics/MetricsUtil.java

```diff
@@ -70,6 +70,14 @@ public class MetricsUtil {
    */
   public static final String PURGED_OFFSET = "purged-offset";
 
+  /**
+   * See {@link com.snowflake.kafka.connector.internal.streaming.TopicPartitionChannel} for offset
+   * description
+   */
+  public static final String OFFSET_PERSISTED_IN_SNOWFLAKE = "persisted-in-snowflake-offset";
+
+  public static final String LATEST_CONSUMER_OFFSET = "latest-consumer-offset";
+
```

Comment:
They're really useful for our internal debugging, do you think they can be send to Snowhouse as well?

Reply:
100% we can send it through the telemetry service, we currently don't have any telemetry coming out streaming specifically (similar to pipe_usage). planning to add it in a follow up pr if i have time. [jira](https://snowflakecomputing.atlassian.net/browse/SNOW-899866)